composer/profiler/profiler_hparams.py

```diff
@@ -8,21 +8,31 @@
 import yahp as hp
 
 from composer.callbacks import CallbackHparams
-from composer.core.profiler import Profiler, ProfilerEventHandlerHparams
-from composer.core.state import State
+from composer.profiler import Profiler, ProfilerEventHandler
 
 if TYPE_CHECKING:
-    from composer.profiler.dataloader_profiler import DataloaderProfiler
     from composer.profiler.json_trace import JSONTraceHandler
-    from composer.profiler.system_profiler import SystemProfiler
-    from composer.profiler.torch_profiler import TorchProfiler
 
 
 @dataclasses.dataclass
 class ProfilerCallbackHparams(CallbackHparams, abc.ABC):
```

Comment:
I don't think this class is being used by anything anymore?

Reply:
(I think we used this when each callback had separate hparams, but it looks like these classes are removed below?)